ts/api/backend.ts

```diff
@@ -201,15 +201,15 @@ export function BackendClient(
 
   const verificaRptT: GetPaymentInfoT = {
     method: "get",
-    url: ({ rptId }) => `/api/v1/payment-requests/${rptId}`,
+    url: ({ rptId, test }) => `/api/v1/payment-requests/${rptId}?${test}`,
     headers: tokenHeaderProducer,
     query: _ => ({}),
     response_decoder: getPaymentInfoDefaultDecoder()
   };
 
   const attivaRptT: ActivatePaymentT = {
     method: "post",
-    url: () => "/api/v1/payment-activations",
+    url: ({ test }) => `/api/v1/payment-activations/${test}`,
```

Comment:
same as above + do not append a `/`

```suggestion
    url: ({ test }) => `/api/v1/payment-activations?test=${test}`,
```